osquery/config/parsers/options.cpp

```diff
@@ -80,6 +80,11 @@ Status OptionsConfigParserPlugin::update(const std::string& source,
       value = std::to_string(option.value.GetInt());
     } else if (option.value.IsNumber()) {
       value = std::to_string(option.value.GetUint64());
+    } else if (option.value.IsObject() || option.value.IsArray()){
+      auto doc = JSON::newFromValue(option.value);
+      doc.toString(value);
+    } else {
+      LOG(WARNING) << "Cannot parse unknown value type for option: " << name;
```

Comment:
IMHO, misspelled option value is good reason to print ERROR message to log

Reply:
Even a config file that cannot be parsed at all gives you just a warning, so I think WARNING is appropriate compared to other parsing related messages